servicetalk-traffic-resilience-http/build.gradle

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright © 2024 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
+
+dependencies {
+  implementation platform(project(":servicetalk-dependencies"))
+
+  api project(":servicetalk-capacity-limiter-api")
+  api project(":servicetalk-circuit-breaker-api")
+  api project(":servicetalk-http-api")
+  api project(":servicetalk-http-utils")
+  api project(":servicetalk-http-netty")
```

Comment:
This is quite unfortunate that this module has to declare `http-netty` as `api` dependency only for the purpose of using `RetryingHttpRequesterFilter.DelayedRetry` interface. Should we consider deprecating it and creating a similar one in `client-api`?

Reply:
Addressed in https://github.com/apple/servicetalk/pull/2927